cumulus/policy.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_sns_topic_policy" "metrics_sns_policy" {
+  for_each = var.metrics_es_aws_account_id != null ? {
+      "granules"    = module.cumulus.report_granules_sns_topic_arn
+      "executions"  = module.cumulus.report_executions_sns_topic_arn
+      "collections" = module.cumulus.report_collections_sns_topic_arn
+      "pdrs"        = module.cumulus.report_pdrs_sns_topic_arn
+  } : {}
+  arn = each.value
+
+  policy = data.aws_iam_policy_document.metrics_sns_topics_policy[each.key].json
+}
+
+data "aws_iam_policy_document" "metrics_sns_topics_policy" {
+  policy_id = "__default_policy_ID"
```

Comment:
Is this id a magic value or something? Should it have the stack prefix in it?

Reply:
As far as I could tell it's a value that the sns access policy has. You can not delete the policy from the topic, only destroy the topic which is prefixed by cumulus.

Reply:
Looks like it is a magic thing for SNS. The examples on the provider and in the AWS docs are all using it. Although I can not seem to find concrete words that say it is.